todo/todo.js

```diff
@@ -44,4 +44,28 @@ export function toggleTodo(todoId) {
     } else {
         alert('task does not exist');
     }
+}
+
+
+//toggles the completed boolean in todos array in user object
+export function nukeTodo(todoId) {
```

Comment:
Nice work figuring out the functionality to remove a to-do from the page and localstorage!